modules/student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name: 'Unknown', parent_permission: true, classroom: 'Unknown')
```

Comment:
- [ ] Please as mentioned by the previous reviewer the `classroom` doesn’t have a default value, kindly remove the "Unknown" default value 🙃 🙃